test/integration/connect/envoy/case-cross-peers-http-router/primary/verify.bats

```diff
@@ -51,7 +51,7 @@ load helpers
 }
 
 @test "s1 upstream should have healthy endpoints for s2" {
-  assert_upstream_has_endpoints_in_status 127.0.0.1:19000 s2.default.default.alpha-to-primary.external HEALTHY 1
+  assert_upstream_has_endpoints_in_status 127.0.0.1:19000 s2.default.primary-to-alpha.external HEALTHY 1
```

Comment:
Note that the new clusterName flips alpha-to-primary -> primary-to-alpha. This is correct since s1 is in the primary cluster